### ISSUE TITLE
Add a image converter like Artwork for DMD do

### ISSUE THREAD

**Issue by goldorakiller** [DMDTools/DOF2DMD#9]
For example with a Retrobat usage, if we want to display the logos systems on the DMD.
- Converts images to a suitable format for DMD displays (128x32, high contrast, black background, centered)

**Comment by gustavoalara** [comment]
See https://github.com/DMDTools/artwork4DMD

**Comment by ojacques** [comment]
I can understand the need. Intent is to do it from within DOF2DMD, and not do it with an "offline tool" like artwork4DMD. Converting takes a bit of time, but we can try that out.

**Comment by goldorakiller** [comment]
Yes convert take a bit of time but we can add a line "autoconvert" in the ini settings ans the file is converted one time and then save in png like artwork does

**Comment by gustavoalara** [comment]
Currently DOF2DMD has the capability to resize images, gif or videos to the DMD size at-the-fly if the source image\gif\video is not too big
